### ISSUE TITLE
[Feature]: Unfloat minimized player

### ISSUE THREAD

**Issue by Jak2k** [fast4x/RiMusic#2038]
### Description

Since the last update, the player is floating.

### Suggested Solution

Add option to revert back to old layout.

**Comment by dAtA-TRoN** [comment]
And where is the issue?

**Comment by aneesh1122** [comment]
Dude...fast4x worked so hard in changing things. There is only so much you can make optional while developing an app.

**Comment by dAtA-TRoN** [comment]
> Dude...fast4x worked so hard in changing things. There is only so much you can make optional while developing an app.

Stay calm. No need to go on the offensive right away. Give him a chance to explain first.

**Comment by fast4x** [comment]
Quiet please...

**Comment by fast4x** [comment]
Player isn't floating, you can set on top or bottom place, but it doesn't moved because are fixed positions.

What do you dislike exactly?

**Comment by fast4x** [comment]
Perhaps you would like the mini player stretched to the edges of the screen as it was before?

**Comment by Jak2k** [comment]
> Perhaps you would like the mini player stretched to the edges of the screen as it was before?

Exactly that. (Maybe it could also be the default for the ViMusic interface option.)

**Comment by fast4x** [comment]
@aneesh1122 @dAtA-TRoN What do you think about options...

**Comment by dAtA-TRoN** [comment]
> What do you think about options...

I like options 🤷🏻‍♂️.

**Comment by aneesh1122** [comment]
> @aneesh1122 @dAtA-TRoN What do you think about options... 

Having too many options will spoil the easiness of this app.

I think all the old UI elements like essential controls, old mini player and navigation bar should be in ViMusic mode. I've seen that there are people who like simple things.

The new appearance changes should be in RiMusic mode. Those who want old experience will use ViMusic.

**Comment by dAtA-TRoN** [comment]
> Having too many options will spoil the easiness of this app.
> 
> I think all the old UI elements like essential controls, old mini player and navigation bar should be in ViMusic mode. I've seen that there are people who like simple things.
> 
> The new appearance changes should be in RiMusic mode. Those who want old experience will use ViMusic.
> 
> Still please listen to the opinions of others as well.

Sure, just include an Expert Mode in the settings, and that will be sufficient. People like you can stick with simplicity, while people like me, who enjoy tinkering with settings, can activate the Expert Mode.

**Comment by fast4x** [comment]
You are right both, there will be to make the settings easier and the game will be done.

**Comment by aneesh1122** [comment]
Fast4x after some people don't like the new changes

![spider-man-norman-osborn.gif](https://github.com/fast4x/RiMusic/assets/45353488/f83e035e-ebb9-42ca-8711-8075a56e7520)

**Comment by fast4x** [comment]
always someone will say "i don't like it" 🥳

**Comment by dAtA-TRoN** [comment]
> always someone will say \"i don't like it\" 🥳


He has accused me of this many times 🫣.

**Comment by NikunjKhangwal** [comment]
> Having too many options will spoil the easiness of this app.
> 
> I think all the old UI elements like essential controls, old mini player and navigation bar should be in ViMusic mode. I've seen that there are people who like simple things.
> 
> The new appearance changes should be in RiMusic mode. Those who want old experience will use ViMusic.
> 
> Still please listen to the opinions of others as well.

The easiness is already gone long ago. It's more of a app for enthusiasts now who like as many options as possible. Like datatron and me.
While your point is legit, i can bet a lot of people will dislike new changes in RiMusic style as well. There's also people who like having mix of both and there's nothing we can complain about because everyone have different taste.

**Comment by ikanakova** [comment]
> It's more of a app for enthusiasts now who like as many options as possible. Like datatron and me.

And me! 😃

**Comment by ikanakova** [comment]
As @dAtA-TRoN wrote in #2045 when all the features in RiMusic are complete, then fast4x can focus on simplifying the app - that Expert mode would be really cool. Users who like simplicity will have it, and we'll turn on Expert Mode 😃 But even if that never happens, what I like about RiMusic is that it's still in development, even if maybe I don't use the features at all.

**Comment by NikunjKhangwal** [comment]
I agree an "expert mode" does sound like a good idea. Although I'll change the wording to "pro mode" lol. Hopefully there'll be voting or some method to determine what features to show in simple mode and what to hide. Chosen by people who really want simplicity and not nerds like us.

**Comment by aneesh1122** [comment]
At this rate this app will end up becoming bigger than AAA games 😂

**Comment by NikunjKhangwal** [comment]
Let's make it AAAA
We're inventing a new class 💪🏼

**Comment by fast4x** [comment]
> > always someone will say \"i don't like it\" 🥳
> 
> 
> He has accused me of this many times 🫣.

I don’t think so, I’m talking about new users, not "old experts" like we all... 😂

**Comment by fast4x** [comment]
> > It's more of a app for enthusiasts now who like as many options as possible. Like datatron and me.
> 
> And me! 😃

I don’t think so, I’m talking about new users, not "old experts" like we all... 😂

**Comment by fast4x** [comment]
> I agree an "expert mode" does sound like a good idea. Although I'll change the wording to "pro mode" lol. Hopefully there'll be voting or some method to determine what features to show in simple mode and what to hide. Chosen by people who really want simplicity and not nerds like us.

Pro is very commercial term... Expert is good...

**Comment by fast4x** [comment]
#2049
Discussion created... 😊
